api-specs/api_spec_PGS.yaml

```diff
@@ -689,8 +689,8 @@ paths:
               $ref: '#/components/schemas/CreditCardResumeRequest'
         required: true
       responses:
-        "302":
-          description: "FOUND, Redirect to url"
+        "200":
+          description: "OK"
```

Comment:
Can you put a more descriptive explanation of what a 200 response entails?

Reply:
Done